code/modules/overmap/spacetravel.dm

```diff
@@ -67,6 +74,8 @@ proc/overmap_spacetravel(var/turf/space/T, var/atom/movable/A)
 	var/ny = 1
 	var/nz = 1
 
+	//Figure out what edge of the world we're off
+		//Todo: Replace this shit
```

Comment:
Have you replaced this shit? Also, recommending we use professional verbage.

Reply:
all todos in this file are for phase 3